content/articles/spotting-of-passing-objects-using-python/index.md

```diff
@@ -0,0 +1,165 @@
+
+
+
+### Prerequisites
+1. You should be an average-level programmer with basic coding skills in python.
+2. A brief overview of and prior knowledge to be equipped with before proceeding will be of great benefit. Following is the link directing you. [How to Adjust Image to Higher or Lower Resolution using Python](/engineering-education/how-to-adjust-image-to-higher-or-lower-resolution-using-python/)
+
+## Creation of the spotting of passing objects
+### Downloading and installing of external libraries
+### Step I:		
+First and foremost, we have to get **OpenCV** in place to be able to manipulate and work with our webcam. Also being a component used in the processing of pictures and live videos  giving solutions to computer vision rising issues.
+In your working environment, open the terminal and paste the command below to have the OpenCV downloaded and installed simultaneously.
+```bash
+ pip install OpenCV-Python
+```
+This library allows for modules such as cv2 to be called and used since cv2 will be playing a key role in our application.
+
+ ### Step II:
+ The second library to download and install will be **numpy**. Having interacted already with OpenCV and known its functions,  it takes the place of an image. So the numpy comes in since it is used in the expression of multi-dimensional arrays. For our case, our representation of the arrays will be the pixels. Likewise, as we had performed for the previous command, we will also be carrying it out same for numpy.
+ ```bash
+ pip install numpy
+ ```
+ ### Step III:
+ Lastly, we are going to install  **imutils** which is useful in  rotating, resizing, and skeletonizing images alongside OpenCV.
+ As you have been following, let's repeat the procedure in the above commands.
+ ```bash
+  pip install imutils
+ ```
+After successful installations, we will now jump into the code area.
+### Case 1:Importing the external libraries.
+As we had discussed earlier the functions of these libraries, we will be now importing them for use in our program.
+```python
+import cv2  
+import numpy as np  
+import datetime  
+import imutils
+```
+Datetime module is called, as it will be used for returning the live or real-time time and date in our footage.
+
+### Case 2:Variables initialization
+This is achieved by setting a variable equal to a given value, thus the variable is created. So in our case, we will be able to use the variables just by calling them.
+```python
+rec=cv2.VideoCapture(0)  
+sto, mapping1=rec.read()  
+sto, mapping2=rec.read()
+```
+In the function VideoCapture, we use `0` since it is the default value for the webcam.
+### Case 3:Detection and conversion of the frame to grayscale
+Video is a series of continuous images with the difference in time. Likewise, it's a slide show of images. So we will have to initially identify and recognize the frames then have them processed. The frame is converted to grayscale to increase the accuracy of key features detection, eliminating possible misses.
```

Comment:
Some readers might not be aware of grayscale. Explain and add any references for it.

Reply:
How does it help in easier image processing? State the factual reasons. (You may even write a separate topic on this)